baseplate/integration/pyramid.py

```diff
@@ -43,9 +58,14 @@ def _on_new_request(self, event):
             # TODO: some metric for 404s would be good
             return
 
-        trace_id = request.headers.get("X-Trace", "no-trace")
-        parent_id = request.headers.get("X-Parent", "no-parent")
-        span_id = request.headers.get("X-Span", "no-span")
+        if self.trust_trace_headers:
+            trace_id = request.headers.get("X-Trace", "no-trace")
```

Comment:
Not really sure, but would we want to make our headers for this slightly more unique? i.e. X-Reddit-Trace or something? Just in case there's something else out there that does tracing and would stomp on others. This might not be a real concern, so who knows!